src/Distribution/Server/Framework/BackupRestore.hs

```diff
@@ -108,9 +108,9 @@ abstractRestoreBackup putSt = go
   where
     go RestoreBackup {..} = AbstractRestoreBackup {
         abstractRestoreEntry = \store entry ->
-          liftM go    <$> (runRestore store $ restoreEntry entry)
+          fmap go    <$> runRestore store (restoreEntry entry)
```

Comment:
In such situations, I use `do`:
```
fmap go <$> do runRestore store $ restoreEntry entry
```
(I don't think hlint knows that trick.)

Reply:
Neat :smiley: applied